device-types/Cisco/Meraki-MR53E.yaml

```diff
@@ -2,7 +2,6 @@
 manufacturer: Cisco
 model: MR53E
 slug: MR53E
-u_height: 0
```

Comment:
These are not rack mountable to my knowledge. We should leave `u_height: 0` defined